### PR TITLE
[Inductor UT] Generalize device-bias code introduced from #136472

### PR DESCRIPTION
**Pull request by anmyachev**
cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang

### CODE REVIEW

File: test/inductor/test_triton_heuristics.py [hunk 1]
```diff
@@ -145,6 +145,7 @@ def pre_hook(kwargs):
         with self.assertRaisesRegex(AssertionError, "pre_hook"):
             autotuner = CachingAutotuner(**args)
 
+    @skipIfXpu
```
Comment (anmyachev) [inline test/inductor/test_triton_heuristics.py:148]:
```suggestion
```

File: test/inductor/test_triton_heuristics.py [hunk 2]
```diff
@@ -145,6 +145,7 @@ def pre_hook(kwargs):
...
     def test_autotune_hints_to_configs(self):
         device_props = DeviceProperties.create(torch.device("cuda"))
```
Comment (anmyachev) [inline test/inductor/test_triton_heuristics.py:150]:
```suggestion
        device_props = DeviceProperties.create(torch.device(GPU_TYPE))
```

**Review by EikanWang — APPROVED** [review]

**Comment by EikanWang** [comment]
Duplicated with https://github.com/pytorch/pytorch/pull/136834

**Comment by anmyachev** [comment]
Closed as duplicated

**Pull request closed without merging**